stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -95,13 +95,15 @@ function Base.getproperty(c::Completion, name::Symbol)
     return getfield(c, name)
 end
 
+representable_string(s::Symbol) = sprint(Base.show_sym, s)
```

Comment:
Feels like this can just be manually inlined? The current function name is a bit strange and inlining it avoids having to come up with a better one, heh.

Reply:
A better name: `Base.sym_to_string`? lol